### PR TITLE
Gidk 23ghp7 report test

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Review by aditi-tiwari17 — APPROVED** [review]

File: apps/web-giddh/src/app/reports/components/columnar-report-table-component/columnar.report.table.component.html [hunk 1]
```diff
@@ -19,27 +19,55 @@
...
                             {{columnarReportResponse?.closingBalance[item.accountUniqueName]?.type
                             ==='DEBIT'? localeData?.dr: localeData?.cr}} </span>
                         </td>
                         <!--Grand  Total -->
                         <td [attr.data-title]="localeData.grand_total" class="text-center">
-                            <span *ngIf="isCrDrChecked && columnarReportResponse?.grandTotal[item.accountUniqueName]?.amount > 0"> {{(columnarReportResponse?.grandTotal[item.accountUniqueName]?.type ==='DEBIT')?'-':''}}</span> {{columnarReportResponse?.grandTotal[item.accountUniqueName]?.amount}}
+                            <span *ngIf="isCrDrChecked && columnarReportResponse?.grandTotal[item.accountUniqueName]?.amount > 0"> {{(columnarReportResponse?.grandTotal[item.accountUniqueName]?.type ==='DEBIT')?'-':''}}</span>
+                            <!-- {{columnarReportResponse?.grandTotal[item.accountUniqueName]?.amount}} -->
+
+                            <amount-field [amount]="columnarReportResponse?.closingBalance[item.accountUniqueName]?.amount" [currencySymbol]="false"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/columnar-report-table-component/columnar.report.table.component.html:52]:
```suggestion
                            <amount-field [amount]="columnarReportResponse?.grandTotal[item.accountUniqueName]?.amount" [currencySymbol]="false"
```

File: apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.html [hunk 2]
```diff
@@ -319,70 +343,89 @@ <h1>{{commonLocaleData?.app_search_suggestion}}</h1>
                     <div class="sale-register-box text-right">
                         <p>{{commonLocaleData?.app_purchase}}</p>
                         <h4>
-                            {{
+                            <!-- {{
                                 (purchaseRegisteDetailedResponse$ | async).debitTotal
                                     | giddhCurrency
-                            }}
+                            }} -->
+                            <amount-field [amount]="(purchaseRegisteDetailedResponse$ | async).debitTotal" [currencySymbol]="false"          [currencyCode]="false">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.html:350]:
```suggestion
                            <amount-field [amount]="(purchaseRegisteDetailedResponse$ | async).debitTotal" [currencySymbol]="false" [currencyCode]="false">
```

File: apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.html [hunk 3]
```diff
@@ -126,9 +126,19 @@
...
+                                <!-- {{ reverseCharge.taxRate }}% -->
+                                <amount-field [amount]="reverseCharge.taxRate"
+                                    [currencySymbol]="false"
+                                    [currencyCode]="false">
+                                </amount-field>
+                            </td>
+                            <td [attr.data-title]="commonLocaleData?.app_tax_amount">
+                                <!-- {{activeCompany.baseCurrencySymbol}}
+                                {{ reverseCharge.taxAmount | giddhCurrency }} -->
+                                <amount-field [amount]="reverseCharge.taxAmount" [currencySymbol]="activeCompany.baseCurrencySymbol" [currencyCode]="false">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.html:139]: Pass currency code

File: apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.html [hunk 4]
```diff
@@ -157,12 +167,32 @@
...
                     <h3>{{localeData?.total_taxable_value}} </h3>
-                    <h1>{{activeCompany?.baseCurrencySymbol}}
-                        {{reverseChargeReportResults?.totalTaxableValue | giddhCurrency }}<span>/-</span></h1>
+                    <h1 class="d-inline-flex">
+                        <!-- {{activeCompany?.baseCurrencySymbol}}
+                        {{reverseChargeReportResults?.totalTaxableValue | giddhCurrency }} -->
+                        <span>
+                            <amount-field [amount]="reverseChargeReportResults?.totalTaxableValue"
+                                [currencySymbol]="activeCompany?.baseCurrencySymbol"
+                                [currencyCode]="false">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.html:176]: Pass currency code

File: apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.html [hunk 5]
```diff
@@ -157,12 +167,32 @@
...
                 <div class="col-sm-6 total-due-sale">
                     <h3>{{localeData?.total_tax_amount}}</h3>
-                    <h1>{{activeCompany?.baseCurrencySymbol}} {{reverseChargeReportResults?.totalTaxAmount | giddhCurrency}}<span>/-</span>
+                    <h1 class="d-inline-flex">
+                        <!-- {{activeCompany?.baseCurrencySymbol}}
+                        {{reverseChargeReportResults?.totalTaxAmount | giddhCurrency}}<span>/-</span> -->
+
+                        <span>
+                            <amount-field [amount]="reverseChargeReportResults?.totalTaxAmount"
+                                [currencySymbol]="activeCompany?.baseCurrencySymbol" [currencyCode]="false">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.html:190]: Pass currency code

File: apps/web-giddh/src/app/vat-report/transactions/vatReportTransactions.component.html [hunk 6]
```diff
@@ -25,7 +25,13 @@
                             <td>{{ (item.voucherDate) ? item.voucherDate : '-' }}</td>
                             <td><a href="javascript:;" [ngClass]="{'noclick': !item.voucherNumber && item.voucherType !== 'purchase'}" (click)="onSelectInvoice(item)">{{ (item.voucherNumber) ? item.voucherNumber : '-' }}</a></td>
                             <td>{{ (item.customerName) ? item.customerName : '-' }}</td>
-                            <td class="text-right">{{activeCompany.baseCurrencySymbol}} {{ item.taxableAmount | giddhCurrency }}</td>
+                            <td class="text-right">
+                                {{activeCompany.baseCurrencySymbol}} {{ item.taxableAmount | giddhCurrency }}
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/vat-report/transactions/vatReportTransactions.component.html:29]: remove this

File: apps/web-giddh/src/app/vat-report/transactions/vatReportTransactions.component.html [hunk 7]
```diff
@@ -25,7 +25,13 @@
...
+                                <amount-field [amount]="item.taxableAmount"
+                                    [currencySymbol]="activeCompany.baseCurrencySymbol"
+                                    [currencyCode]="invFormData.accountDetails?.currency?.code">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/vat-report/transactions/vatReportTransactions.component.html:32]: is invFormData part of vat report transactions?

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

**Pull request closed without merging**